### PR TITLE
Q10325_SearchingForNewPower

### PR DESCRIPTION
**Pull request by Neanrakyr**
Quest Q10325_SearchingForNewPower

### CODE REVIEW

File: L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/32148-01.htm [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<html><head><body>
```
Comment (Sdw-) [inline L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/32148-01.htm]: put the npc name and br on same line

File: L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/Q10325_SearchingForNewPower.java [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+		{
+			return null;
+		}
+		
+		String htmltext = null;
+		switch (event)
+		{
+			case "quest_accept":
+			{
+				if (player.getRace() == Race.HUMAN)
```
Comment (Sdw-) [inline L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/Q10325_SearchingForNewPower.java]: Use a switch on player.getRace() here

File: L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/Q10325_SearchingForNewPower.java [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+	{
+		final QuestState qs = getQuestState(player, true);
+		String htmltext = getNoQuestMsg(player);
+		
+		if (qs.isCompleted())
+		{
+			htmltext = getAlreadyCompletedMsg(player);
+		}
+		
+		final QuestState quest = player.getQuestState(Q10324_FindingMagisterGallint.class.getSimpleName());
```
Comment (Sdw-) [inline L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/Q10325_SearchingForNewPower.java]: Please use 

addCondCompletedQuest(Q10324_FindingMagisterGallint.class.getSimpleName(), "32980-noQuestComplete.htm");

and drop those check

File: L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/Q10325_SearchingForNewPower.java [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+		if (!quest.isCompleted())
+		{
+			return "32980-noQuestComplete.htm";
+		}
+		
+		switch (npc.getId())
+		{
+			case GALLINT:
+				if (qs.isCreated())
```
Comment (Sdw-) [inline L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/Q10325_SearchingForNewPower.java]: if / else if / else

File: L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/32980-noLevel.htm [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<html><body>Gallint:<br>
```
Comment (Sdw-) [inline L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/32980-noLevel.htm]: Is this HTML retail ?

File: L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/32980-noQuestComplete.htm [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<html><body>Gallint:<br>
```
Comment (Sdw-) [inline L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/32980-noQuestComplete.htm]: Is this HTML retail ?

File: L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/32980-noErtheia.htm [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<html><body>Gallint:<br>
```
Comment (Sdw-) [inline L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/32980-noErtheia.htm]: Is this HTML retail ?

File: L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/Q10325_SearchingForNewPower.java [hunk 8]
```diff
@@ -0,0 +1,285 @@
...
+		}
+		
+		String htmltext = null;
+		switch (event)
+		{
+			case "quest_accept":
+			{
+				switch(player.getRace())
+				{
+					case HUMAN:
```
Comment (Sdw-) [inline L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/Q10325_SearchingForNewPower.java]: Missing break on each case

File: L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/Q10325_SearchingForNewPower.java [hunk 9]
```diff
@@ -0,0 +1,285 @@
...
+		String htmltext = getNoQuestMsg(player);
+		
+		if (qs.isCompleted())
+		{
+			htmltext = getAlreadyCompletedMsg(player);
+		}
+		
+		switch (npc.getId())
+		{
+			case GALLINT:
```
Comment (Sdw-) [inline L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/Q10325_SearchingForNewPower.java]: Use braket on case

File: L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/Q10325_SearchingForNewPower.java [hunk 10]
```diff
@@ -0,0 +1,305 @@
...
+	private static final int TALBOT = 32156; // 2 - Human
+	private static final int CIDNET = 32148; // 3 - Elven
+	private static final int BLACK = 32161; // 4 - Dark Elven
+	private static final int HERTZ = 32151; // 5 - Orc
+	private static final int KINCAID = 32159; // 6 - Dwarf
+	private static final int XONIA = 32144; // 7 - Kamael
+	
+	// Items
+	private static final ItemHolder SOULSHOTS = new ItemHolder(1835, 1000);
+	private static final ItemHolder SPIRITSHOTS = new ItemHolder(2509, 1000);
```
Comment (jurchiks) [inline L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/Q10325_SearchingForNewPower.java]: There's usually 500 spiritshots/1000 soulshots... Is this value here really correct?

Comment (Neanrakyr) [inline L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/Q10325_SearchingForNewPower.java]: Ok didn't know for Spiritshots. I do the quest has a Humain warrior. So you may be in trust

Comment (Gladicek) [inline L2J_DataPack/dist/game/data/scripts/quests/Q10325_SearchingForNewPower/Q10325_SearchingForNewPower.java]: Well, I have this quest done with all htmls etc. Dont waste your time with it anymore man :D

**Comment by Sdw-** [comment]
@Neanrakyr ,

Thanks for your work on this. As we discussed on gitter, I'll close this PR as @Galdicek already got it working without any custom dialog. Since he based it on your earlier work, he tagged you as author as well.

This is not a waste of your time as I'm sure you learned a bit how the quest engine of L2J works.

From now on, everyone, be sure to declare yourself here : 
http://www.l2jserver.com/forum/viewtopic.php?f=113&t=30647
before starting anything.

**Comment by Sdw-** [comment]
@Neanrakyr ,

Thanks for your work on this. As we discussed on gitter, I'll close this PR as @Gladicek already got it working without any custom dialog. Since he based it on your earlier work, he tagged you as author as well.

This is not a waste of your time as I'm sure you learned a bit how the quest engine of L2J works.

From now on, everyone, be sure to declare yourself here : 
http://www.l2jserver.com/forum/viewtopic.php?f=113&t=30647
before starting anything.

**Pull request closed without merging**